Dockerfile

```diff
@@ -23,6 +23,11 @@ RUN apk update && \
 
 RUN curl -sSL https://apk.cloudposse.com/install.sh | bash
 
+## Install terraform-config-inspect (required for bats tests)
+ENV GO111MODULE="on"
+RUN go get github.com/hashicorp/terraform-config-inspect && \
```

Comment:
We can soon replace this with the package. https://github.com/cloudposse/packages/pull/748